include/CppUTest/UtestMacros.h

```diff
@@ -91,115 +91,157 @@
 
-#define CHECK_LOCATION_TEXT(condition, checkString, conditionString, text, file, line) \
-    { UtestShell::getCurrent()->assertTrueText((condition) != 0, checkString, conditionString, text, file, line); }
+#define CHECK_FALSE_TEXT(condition, text)\
+  CHECK_FALSE_LOCATION(condition, "CHECK_FALSE", #condition, text, __FILE__, __LINE__)
 
-#define CHECK_LOCATION_TRUE(condition, checkString, conditionString, file, line)\
-  { UtestShell::getCurrent()->assertTrue((condition) != 0, checkString, conditionString, file, line); }
+#define CHECK_TRUE_LOCATION(condition, checkString, conditionString, text, file, line)\
+  { UtestShell::getCurrent()->assertTrue((condition) != 0, checkString, conditionString, text, file, line); }
 
-#define CHECK_LOCATION_FALSE(condition, checkString, conditionString, file, line)\
-  { UtestShell::getCurrent()->assertTrue((condition) == 0, checkString, conditionString, file, line); }
+#define CHECK_FALSE_LOCATION(condition, checkString, conditionString, text, file, line)\
+  { UtestShell::getCurrent()->assertTrue((condition) == 0, checkString, conditionString, text, file, line); }
 
 //This check needs the operator!=(), and a StringFrom(YourType) function
-#define CHECK_EQUAL(expected,actual)\
-  CHECK_EQUAL_LOCATION(expected, actual, __FILE__, __LINE__)
+#define CHECK_EQUAL(expected, actual)\
+  CHECK_EQUAL_LOCATION(expected, actual, "", __FILE__, __LINE__)
+
+#define CHECK_EQUAL_TEXT(expected, actual, text)\
+  CHECK_EQUAL_LOCATION(expected, actual, text, __FILE__, __LINE__)
 
-#define CHECK_EQUAL_LOCATION(expected,actual, file, line)\
+#define CHECK_EQUAL_LOCATION(expected, actual, text, file, line)\
   { if ((expected) != (actual)) { \
       if ((actual) != (actual)) \
       	  UtestShell::getCurrent()->print("WARNING:\n\tThe \"Actual Parameter\" parameter is evaluated multiple times resulting in different values.\n\tThus the value in the error message is probably incorrect.", file, line); \
       if ((expected) != (expected)) \
       	  UtestShell::getCurrent()->print("WARNING:\n\tThe \"Expected Parameter\" parameter is evaluated multiple times resulting in different values.\n\tThus the value in the error message is probably incorrect.", file, line); \
-      UtestShell::getCurrent()->assertEquals(true, StringFrom(expected).asCharString(), StringFrom(actual).asCharString(), file, line); \
+      UtestShell::getCurrent()->assertEquals(true, StringFrom(expected).asCharString(), StringFrom(actual).asCharString(), text, file, line); \
   } \
   else \
   { \
-    UtestShell::getCurrent()->assertLongsEqual((long)0, (long)0, file, line); \
+    UtestShell::getCurrent()->assertLongsEqual((long)0, (long)0, "", file, line); \
   } }
 
 //This check checks for char* string equality using strcmp.
 //This makes up for the fact that CHECK_EQUAL only compares the pointers to char*'s
-#define STRCMP_EQUAL(expected,actual)\
-  STRCMP_EQUAL_LOCATION(expected, actual, __FILE__, __LINE__)
+#define STRCMP_EQUAL(expected, actual)\
```

Comment:
Can you, in the future, try to keep the irrelevant changes (such as this additional space) out of the diffs :)